src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+
 public class UserRequestDto {
```

Comment:
use @RepeatPassword annotation

Reply:
so place it over the class
![image](https://user-images.githubusercontent.com/118391929/222444391-5910c287-d24a-4db3-b0ff-d802159af5f9.png)